info/v2/container.go

```diff
@@ -85,7 +85,7 @@ type ContainerSpec struct {
 	Image string `json:"image,omitempty"`
 }
 
-type ContainerStats struct {
+type DeprecatedContainerStats struct {
```

Comment:
I'm fine with splitting it up, but I won't have time to work on it until
Thursday. So we can either merge this in if it's a blocker out wait for two
more days..
On Jan 11, 2016 7:52 PM, "Tim St. Clair" <notifications@github.com> wrote:

> In info/v2/container.go
> <https://github.com/google/cadvisor/pull/1035#discussion_r49413412>:
>
> > @@ -85,7 +85,7 @@ type ContainerSpec struct {
> >  	Image string `json:"image,omitempty"`
> >  }
> >
> > -type ContainerStats struct {
> > +type DeprecatedContainerStats struct {
>
> While I like this change, I wonder if it would be better to split it into
> a separate PR? Both because it seems a little unrelated to the disk stats,
> and we could make sure the disk changes are not blocked while dealing with
> merge conflicts etc. from the API change. If you think it's easier to keep
> it in this PR, I won't stop you...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1035/files#r49413412>.
>